Doc/library/getpass.rst

```diff
@@ -46,7 +46,7 @@ The :mod:`getpass` module provides two functions:
    :envvar:`USER`, :envvar:`!LNAME` and :envvar:`USERNAME`, in order, and
    returns the value of the first one which is set to a non-empty string.  If
    none are set, the login name from the password database is returned on
-   systems which support the :mod:`pwd` module, otherwise, an exception is
-   raised.
+   systems which support the :mod:`pwd` module, otherwise, an :exc:`OSError`
+   is raised.
```

Comment:
Lets add a versionchanged 3.13 marker mentioning that a variety of inconsistent exceptions other than just OSError were raised in previous versions.